packages/backend/serverless.yml

```
@@ -79,6 +79,12 @@ functions:
       - http:
           path: create-order
           method: post
+  get-order-data:
+    handler: get-order-count.getOrderCount
+    events:
+      - http:
+          path: /get-order-count
```

Логичнее назвать это в множественном числе: order**s**
```suggestion
          path: /get-orders-count
```

---

И название файла тогда тоже надо будет заменить

---

Давай назовем файл `get-orders-count_handler.js`, чтоб точно можно было сказать по имени, что это обработчик